stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -78,7 +79,7 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
     type(block_control_type), intent(inout) :: Atm_block
     integer,                  intent(out)   :: ierr
 
-    integer :: nthreads, nb, levs, maxblk, nblks
+    integer :: nthreads, nb, levs, maxblk, nblks, v
```

Comment:
Integer variables are usually i,j,k,l,m,n - can you use a more common name than `v` please. This convention comes from the old days where implicit variables where integers when between i and n, but reals otherwise (https://docs.oracle.com/cd/E19957-01/805-4939/z40007365fbc/index.html).

Reply:
Would "var" be allowed here instead of "v"?

Reply:
Yes, that provides more meaning than just one letter. If one letter representing an integer, I would use `n`.

Reply:
OK, given it's an integer, I changed it to "n."